src/ui/filter/security.go

```diff
@@ -78,6 +113,11 @@ func SecurityFilter(ctx *beegoctx.Context) {
 		return
 	}
 
+	if !filterReq(req) {
```

Comment:
As discussed, we don't need this filter for now as the ReqCtxModifier has been doing the filter already, we'll consider optimize when we see a perf issue.